lib/play.js

```diff
@@ -0,0 +1,42 @@
+const { getTeamChoice } = require("./chooseTeam");
+const {
+  setTeamChoice,
+  setUserGridChoice,
+  setCpuGridChoice,
+} = require("./store");
+const { pickOrder, playOrder } = require("./playOrder");
+const logGrid = require("./logGrid");
+const { getUserGridChoice } = require("./chooseUserGrid");
+const { getCpuGridChoice } = require("./chooseCpuGrid");
+const { checkForGameOver } = require("./checkWin");
+
+const userTurn = async (store) => {
+  setUserGridChoice(store, await getUserGridChoice(store));
+  if (!checkForGameOver(store)) {
+    setCpuGridChoice(store, await getCpuGridChoice(store));
+  }
+  logGrid(store);
+};
+
+const cpuTurn = async (store) => {
+  setCpuGridChoice(store, await getCpuGridChoice(store));
+  logGrid(store);
+  if (!checkForGameOver(store)) {
+    setUserGridChoice(store, await getUserGridChoice(store));
+  }
+};
+
+const play = async (store) => {
+  setTeamChoice(store, await getTeamChoice());
+  playOrder(store, pickOrder());
+  logGrid(store);
+  while (store.winner === false) {
+    if (store.firstPlayer === "user") {
+      await userTurn(store);
+    } else {
+      await cpuTurn(store);
+    }
```

Comment:
[suggested refactor for another PR?] I was confused by this at first.

userTurn() does 
// make a user turn
// check for winner
// make a cpu turn

and cpuTurn() does
// make a cpu turn
// check for winner
// make a user turn

and then this for loop also does
// check for winner

This means that if we unroll the loop for `store.firstPlayer === "user"`, we get:

/// first loop
// check for winner
// make a user turn
// check for winner
// make a cpu turn
/// next loop
// check for winner
// make a user turn
// check for winner
// make a cpu turn
/// next loop

This is doing the correct thing, but isn't how I would expect it to be written

If you keep track of some kind of `store.nextPlayer` or something, then you might be able to simplify userTurn()/cpuTurn() to just `// make a user turn`/`// make a cpu turn`. It would then be:

/// first loop
// check for winner
// make a user turn
/// next loop
// check for winner
// make a cpu turn
/// next loop